src/data/equiv/ring.lean

```diff
@@ -170,6 +180,10 @@ instance has_coe_to_ring_hom : has_coe (R ≃+* S) (R →+* S) := ⟨ring_equiv.
 @[norm_cast] lemma coe_ring_hom (f : R ≃+* S) (a : R) :
   (f : R →+* S) a = f a := rfl
 
+lemma coe_ring_hom_inj {R S : Type*} [semiring R] [semiring S] (f g : R ≃+* S) :
+  f = g ↔ (f : R →+* S) = g :=
```

Comment:
Aren't these kinds of statement called `_inj_iff`?

Reply:
Yep, thanks for catching it.